src/main/java/frc/robot/subsystems/Loading.java

```diff
@@ -0,0 +1,79 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.LoadingConstants;
+
+public class Loading extends SubsystemBase {
+  /**
+   * Creates a new Loading.
+   */
+
+  private final CANSparkMax m_loadingMotor;
+  private final CANSparkMax m_cellevatorMotor1;
+  private final CANSparkMax m_cellevatorMotor2;
+  private final DigitalInput m_beamTop;
+  private final DigitalInput m_beamBottom;
+
+
+  public Loading() {
+
+    m_loadingMotor = new CANSparkMax(LoadingConstants.kLoadingMotorPort, MotorType.kBrushed);
+    m_cellevatorMotor1 = new CANSparkMax(LoadingConstants.kLowCellevatorMotorPort, MotorType.kBrushed);
+    m_cellevatorMotor2 = new CANSparkMax(LoadingConstants.kHighCellevatorMotor2Port, MotorType.kBrushed);
+    m_beamTop = new DigitalInput(LoadingConstants.kBeamBreakOutput1Port);
+    m_beamBottom = new DigitalInput(LoadingConstants.kBeamBreakOutput2Port);
+
+    motorInit(m_loadingMotor, false);
+    motorInit(m_cellevatorMotor1, false);
+    motorInit(m_cellevatorMotor2, false);
```

Comment:
If we're not planning on inverting any of the motors, let's cut that out of the motorInit entirely.  If we want to keep it, then the inversion boolean should be a constant stored in the Constants.

Reply:
the invert parameter was taken out